src/components/profile/index.js

```diff
@@ -83,16 +108,18 @@ export default class SignUp extends React.Component {
   }
 
   handleSignUp = async () => {
-    const { signUp, history } = this.props
+    const { signUp, history }: ProfilePropsType = this.props
     const { email, password } = this.state
 
+    // $FlowFixMe Cannot call signUp because undefined [1] is not a function.
```

Comment:
Line 22 - Signup prop is optional and if undefined this call will throw

Reply:
[aaron/feature/profile-scene a05ea9a] :ok_hand: [profile-scene] signUp prop is not optional